### DISCUSSION TITLE
How should I configure AutoHotkey to be able to launch the program from the script?

### DISCUSSION THREAD

**Discussion by Blu3train** [buliasz/tesstrain-windows-gui#19] [Q&A] [answered]
I state that I have never used AutoHotkey.

I would like to be able to run the gui from the script to do some tests with customizations, specifically I wanted to do some tests to see if I can put a system sound at the end of training more evident than the one you kindly added.
In this way, instead of always asking, I could contribute with proposals for small changes, as long as I can do something decent.

I installed AutoHotkey leaving the proposed defaults, but when I run the script or try to compile it, I get the following error

This is if I run as script:
![image](https://user-images.githubusercontent.com/80602322/190847721-76c25b82-6ed1-4ff5-9ed8-48961eac76d0.png)

This is if I compile the script:
![image](https://user-images.githubusercontent.com/80602322/190847746-f781e3b3-af08-4e4d-8f9c-ba721bb426f1.png)

It's basically the same error.

What am I doing wrong? 
Thank you

**Comment by buliasz** [comment, marked as answer]
I'm guessing you have downloaded AutoHotKey in version 1. Those scripts use AutoHotKey version 2 (which is still in beta). You can find it [here](https://www.autohotkey.com/v2/).

**Comment by Blu3train** [comment]
Thank you, it works now.

**Comment by buliasz** [comment]
You may want to install syntax for Notepad++ (if you use it): https://www.autohotkey.com/boards/viewtopic.php?t=72424
